CLAUDE.md

```diff
@@ -0,0 +1,212 @@
+# CLAUDE.md
```

Comment:
There is memory bank for the project already created, lets re-use or have a section there itself, instead having this file